tests/test_level_18.py

```diff
@@ -229,4 +231,16 @@ def test_var_undefined_error_message(self):
 
       # deze extra check functie kan nu niet mee omdat die altijd op result werkt
       # evt toch splitsen in 2 (pos en neg?)
-      # self.assertEqual('name', context.exception.arguments['name'])
\ No newline at end of file
+      # self.assertEqual('name', context.exception.arguments['name'])
+    
+    def test_input_without_argument(self):
+      self.multi_level_tester(
+        code="name is input",
+        exception=hedy.exceptions.IncompleteCommandException
+      )
+
+    def test_input_without_text_inside(self):
```

Comment:
But here I feel we should allow this, because Python allows it too. It is not really pretty and kind of confusing but yeah we are getting close to Python now so I am inclined to allow this. What do you think?

Reply:
Yes! I had a lot of doubts with that one and actually wanted to ask you. It's a bit tricky since there is not a direct way of differenciate between an input with no brackets and one with brackets and nothing inside, maybe add a new error rule for the ,`x is input` in any case I'll solve it and update this :)

Also, the same thing happens with print(), gives a parse error, I assume you also want that to be valid maybe?

Reply:
Yeah I understand, it *is* a tough call! 

But indeed if we want to fix it, it would require two rules. Do you want to pick it up further in this PR? And yes yes, then do it for print also in one go, it is the same issue.

Reply:
Yes! I want to pick it up further in this PR, give me one day or maybe two :)

Reply:
Sure no rush!! Super grateful for all your hard work!